python/cudf/dataframe/columnops.py

```diff
@@ -93,55 +93,55 @@ def fillna(self, fill_value, inplace):
         raise NotImplementedError
 
 
-def make_null_like(other, size=None, dtype=None):
-    if size is None:
-        size = other.size
-    if dtype is None:
-        dtype = other.dtype
-    mask = cudautils.make_mask(size)
-    cudautils.fill_value(mask, 0)
+def column_empty_like(column, dtype, masked, newsize=None):
+    """Allocate a new column like the given *column*
+    """
+    row_count = len(column) if newsize is None else newsize
+    categories = None
+    if hasattr(column, 'cat'):
```

Comment:
Should check dtype instead of this as it would be safer. Could need a `cat` attribute in the future for custom exceptions or the like.

Reply:
Done